### ISSUE TITLE
E404 error with xdrip+ and Fitbit

### ISSUE THREAD

**Issue by simonish** [Rytiggy/Glance#98]
Everything has been working fine, fitbit app on android seems to have updated overnight and this morning all it shows is E404, I have tried a few other cgm faces and non will communicate so I am suspecting the fitbit app... any advice?

I have tried restarting everything

**Comment by jbuckhi** [comment]
I can confirm this. I observed exactly the same behaviour. The error appeared after yesterdays update of the fitbit app to version 3.9.1 (20243443). In addition I tested to switch the data source to my nightscout instance: no change - same error. By the way: my xDrip+ is unchanged (d7689af-2019.11.02).

**Comment by gitanO0** [comment]
Seems to be 127.0.0.1 not working any more. When I use sgv.json from my nightscout site i get data again.

**Comment by norbertzpb** [comment]
I can confirm this. After Fitbit update E404 Error on Versa2.

**Comment by simonish** [comment]
This is on the Fitbit Versa 1 with Android 10 and Xdrip+

**Comment by jimk9** [comment]
Hi,
I think it may be more than that. I have huawei p20, still on 9 with xfrip+. All has been stable for months. As of last night I am now getting this error. No settings/software updated bar fitbit app. Looks like it updated during the night to version 3.9.1 (20243443).

**Comment by simonish** [comment]
Yes likewise, fully stable until the overnight fitbit app update, now I am (thankfully still have it) using my old ticwatch e until it all works again

**Comment by jimk9** [comment]
Might be worth a look at issue #97 for possible workaround

**Comment by MichelaBelluta** [comment]
Help! I'm keep having the e404 error pn fitbit versa 2 despite trying all the troubleshooting I've found here. Im using xdrip

**Comment by Rytiggy** [comment]
It's a Fitbit app issues, they have yet to confirm exactly what it is but this happened a few years back. They are supposed to allow local http calls to work but accidentally broke it in the last update for Android, they said the next update will fix it.

**Comment by MasterPlexus** [comment]
I tried to use interims my own nightscout instance, unfortunately it looks like the Watchface created [objects] for bgs in the Json response (the original json looks fine... Really sad, that no way is currently possible...
